_includes/benchmarks.html

```diff
@@ -1,30 +1,31 @@
 <table class="benchmarks">
 <colgroup>
-<col class="name">
-<col class="relative" span="11">
+<col class="name"></col>
+<col class="relative" span="12"></col>
```

Comment:
Can we get these without the `</col>` – browsers render them in the output? I made that change to Julia master already, so if you rebase your changes, the Perl script that generates this should not include these.